### PR TITLE
Track video tutorial 'play' event

### PR DESCRIPTION
**Pull request by ezequielc**
I had to add a directive to watch 'playing' event on the view. Ng-click was not being triggered when clicking the play button of the video.
@Rise-Vision/apps Please review. Thanks

### CODE REVIEW

File: web/scripts/launcher/directives/dtv-track-play.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+angular.module('risevision.apps.launcher.directives')
+  .directive('trackPlay', ['launcherTracker',
+    function (launcherTracker) {
+      return {
+        restrict: 'A',
+        link: function ($scope, element, attrs) {
+          element.on('playing', function ($event) {
+            launcherTracker('Played Tutorial')
```
Comment (alex-deaconu) [inline web/scripts/launcher/directives/dtv-track-play.js]: @ezequielc just some linting: missing semi colon here, and the `attrs` parameter not being used. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc just one comment, otherwise code looks good. Also tested and the event is being tracked correctly. Thanks!

**Comment by fjvallarino** [comment]
:+1:

File: web/scripts/launcher/directives/dtv-track-play.js (hunk 1 above)
Comment (ezequielc) [inline web/scripts/launcher/directives/dtv-track-play.js]: @alex-deaconu Good catch. Fixing it. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc looks good, thanks for fixing 👍 !

**Pull request merged**